examples/self-supervised-learning/demo_equivariant_imaging.py

```diff
@@ -180,9 +180,9 @@
 scheduler = torch.optim.lr_scheduler.StepLR(optimizer, step_size=int(epochs * 0.8) + 1)
 
 # start with a pretrained model to reduce training time
-url = online_weights_path() + "demo_ei_ckp_150.pth"
+url = online_weights_path() + "new_demo_ei_ckp_150.pth"
```

Comment:
Because of change in the optimizer's structure (moved from dict to class) percolating through the `state_dict` and making the old checkpoint incompatible with the new structure.